hexrd/findorientations.py

```diff
@@ -388,56 +388,48 @@ def load_eta_ome_maps(cfg, pd, image_series, hkls=None, clean=False):
 
     Parameters
     ----------
-    cfg : TYPE
-        DESCRIPTION.
-    pd : TYPE
-        DESCRIPTION.
-    image_series : TYPE
-        DESCRIPTION.
-    hkls : TYPE, optional
-        DESCRIPTION. The default is None.
-    clean : TYPE, optional
-        DESCRIPTION. The default is False.
+    cfg: Config instance
+        root config file for this problem
+    pd: PlaneData instance
+        crystallographic properties
+    image_series: ImageSeries instance
+        stack of images
+    hkls: list, default = None
+        list of HKLs used in the eta-omega maps.
+    clean: bool, default = False
+        flag indicating whether (if True) to overwrite existing maps file
 
     Returns
     -------
-    TYPE
-        DESCRIPTION.
+    list:
+        list of eta-omega map arrays
 
     """
-    # check maps filename
-    if cfg.find_orientations.orientation_maps.file is None:
-        maps_fname = '_'.join([cfg.analysis_id, "eta-ome_maps.npz"])
+    fn = cfg.find_orientations.orientation_maps.file
+    if clean:
+        logger.info(
+            'clean option specified; recomputing eta/ome orientation maps'
+        )
+        res = generate_eta_ome_maps(cfg, hkls=hkls)
     else:
-        maps_fname = cfg.find_orientations.orientation_maps.file
-
-    fn = os.path.join(cfg.working_dir, maps_fname)
-
-    # ???: necessary?
-    if fn.split('.')[-1] != 'npz':
-        fn = fn + '.npz'
-
-    if not clean:
         try:
-            res = EtaOmeMaps(fn)
+            res = EtaOmeMaps(str(fn))
             pd = res.planeData
-            logger.info('loaded eta/ome orientation maps from %s', fn)
+            logger.info(f'loaded eta/ome orientation maps from {str(fn)}')
             shkls = pd.getHKLs(*res.iHKLList, asStr=True)
             logger.info(
                 'hkls used to generate orientation maps: %s',
                 [f'[{i}]' for i in shkls]
             )
         except (AttributeError, IOError):
-            logger.info("specified maps file '%s' not found "
-                        + "and clean option specified; "
-                        + "recomputing eta/ome orientation maps",
-                        fn)
+            logger.info(
+                f"specified maps file '{str(fn)}' not found "
+                f"and clean option not specified; "
+                f"recomputing eta/ome orientation maps"
+            )
             res = generate_eta_ome_maps(cfg, hkls=hkls)
-    else:
-        logger.info('clean option specified; '
-                    + 'recomputing eta/ome orientation maps')
-        res = generate_eta_ome_maps(cfg, hkls=hkls)
-    filter_maps_if_requested(res, cfg)
+
+            filter_maps_if_requested(res, cfg)
```

Comment:
```suggestion
    filter_maps_if_requested(res, cfg)
```

It looks like this filtering occurred outside of the `if` block, right?

Reply:
Good catch.